DylibBinder/DBAssociatedType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DylibBinder {
+	// TODO Add AssociatedTypes
+	public class DBAssociatedType {
+		public DBAssociatedType (string name)
+		{
+			Name = name;
+		}
+
+		public string Name { get; }
+	}
+
+	public class DBAssociatedTypes {
+		public DBAssociatedTypes ()
+		{
+		}
+
+		public List<DBAssociatedType> AssociatedTypes { get; } = new List<DBAssociatedType> ();
```

Comment:
Same, why value does the class five, can't we just use a List<>. If it only adde the list object, can we just make it extend form list?
```csharp
public class DBAssociatedTypes : List<DBAssociatedType>
```

Reply:
Discussion